crates/fluvio-version-manager/src/command/itself/install.rs

```diff
@@ -17,7 +17,7 @@ case ":${PATH}:" in
     *:"$HOME/.fvm/bin":*)
         ;;
     *)
-        export PATH="$PATH:$HOME/.fvm/bin"
+        export PATH="$PATH:$HOME/.fvm/bin:$HOME/.fluvio/bin"
```

Comment:
This allows the user to have both:

- `~/.fvm/bin`
- `~/.fluvio/bin`

In their PATH after executing `source ~/.fvm/env`